recommenders/models/newsrec/newsrec_utils.py

```diff
@@ -310,15 +310,15 @@ def get_mind_data_set(type):
 
     if type == "large":
         return (
-            "https://mind201910small.blob.core.windows.net/release/",
+            "https://recodatasets.z20.web.core.windows.net/newsrec",
```

Comment:
The trailing `/` was removed, perhaps it was important

Reply:
done